### ISSUE TITLE
feat: support Mobile Touch Gestures

### ISSUE THREAD

**Issue by HansKrywaa** [Mawi137/ngx-image-cropper#272]
Hey there,

is it possible to support mobile Touch Gestures? I want to able to move the crop area while dragging and resize it with pinch to zoom? 🤓

**Comment by Mawi137** [comment]
It possible yes, but not an easy task. Feel free to fork this repo and give it a go :)

**Comment by HansKrywaa** [comment]
Ok i can try 😂 How can i use my local cloned fork in a project to test my changes? I tried:
 - npm run build:lib
- npm run copy:files
- install in my test project from file:../ngx-image-cropper/dist/ngx-image-cropper

I get the following Error:
```
ERROR Error: Uncaught (in promise): NullInjectorError: StaticInjectorError(AppModule)[ImageCropperComponent -> DomSanitizer]: 
  StaticInjectorError(Platform: core)[ImageCropperComponent -> DomSanitizer]: 
    NullInjectorError: No provider for DomSanitizer!
```

**Comment by Mawi137** [comment]
Npm start should launch the demo project

**Comment by HansKrywaa** [comment]
but i have to test on a device for the pinch gesture

**Comment by Mawi137** [comment]
You can do a prod build and then serve the demo app using `http-server` (available on npm). Or you can do a build like you did and then use nom pack to create a tar.gz file and then use `npm install path/to/tar.gz`. Or create a stackblitz with your code.

**Comment by HansKrywaa** [comment]
PR is created - please give it a look 😇

**Comment by narhc8** [comment]
Any idea when this will be added? Looking forward to implement this 👨‍💻

**Comment by HansKrywaa** [comment]
We are currently discussing my PR, so i think it will be added soon 😊

**Comment by HansKrywaa** [comment]
@narhc8 in the meantime you can check out my Fork (https://github.com/HansKrywaa/ngx-image-cropper) and let me know if this works for you 😊 

Steps:
- Clone Fork
- Run `npm run build:lib`
- Run `npm pack ./dist/ngx-image-cropper`

Install the Produced File in your Project

**Comment by Mawi137** [comment]
Released in version `3.1.0`. Thanks @HansKrywaa !
